docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+                        An icon input field can show that it is currently loading data.
+
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputLoading').default}
+                >
+                    <ExampleInputLoading />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Max & Min
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Specifies the minimum and maximum value that the field can have.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputMaxMin').default}
+                >
+                    <ExampleInputMaxMin />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Max Length
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Specifies the maximum number of characters that the user can enter.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputMaxLength').default}
+                >
+                    <ExampleInputMaxLength />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Min Length
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Specifies the minimum number of characters that the user needs to enter.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputMinLength').default}
+                >
+                    <ExampleInputMinLength />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onBlur Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onBlur</code> event from parent.
```

Comment:
Maybe reword this slightly?
```
Will call an <code>onBlur</code> event handler function provided by the parent.
```

Reply:
Fixed. Please check this [Commit](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/46c752aa0e781aedba48ebab9e7a168431118aa2)

Reply:
Looks good.  Resolving.